boards/stm32f3discovery/Makefile.include

```diff
@@ -2,6 +2,9 @@
 export CPU = stm32f3
 export CPU_MODEL = stm32f303vc
 
+# use on-board LEDs
+USEMODULE += onboard_led
```

Comment:
I'm unsure if we really want to make it a default per board, or rather let the developer decide on a per app/tests usage?